packages/server/src/notification/notification.controller.ts

```diff
@@ -37,9 +37,13 @@ export class NotificationController {
     return `registration success for ${body.phoneNumber}`;
   }
 
+  // TODO: GET RID OF THIS
   @Post('/notify_user/:user_id')
   async notifyUser(@Param('user_id') user_id: number): Promise<string> {
-    await this.notifService.notifyUser(user_id);
+    await this.notifService.notifyUser(
+      user_id,
+      "bruh if you're still tryna test notifications, ligma balls, FUCK FUCK FUCK FUCK FUCK, sugondese, hot cheetos",
```

Comment:
this is the test endpoint. we decided to put in all of the "hot words" that we're bound to search for when we do a final cleanup